{
  "version": 3,
  "file": "",
  "sourceRoot": "",
  "sources": [
    "test/exceptions.test.coffee"
  ],
  "names": [],
  "mappings": "AAAwB;AAExB,OAAO,CAAA,OAAP,MAAA;;AACA,MAAM,CAAC,MAAP,CAAc,MAAd,EAAsB,GAAtB;;AACA,OAAA;EAAQ,KAAR;EAAe,QAAf;CAAA,MAAA,6BAJwB;;;AAQxB,KAAA,CAAM,QAAA,CAAA,CAAA;EACL,wBAAA,CAAyB,IAAzB;SACA,KAAA,CAAM,KAAN;AAFK,CAAN;;AAIA,KAAA,CAAM,QAAA,CAAA,CAAA;EACL,wBAAA,CAAyB,IAAzB;SACA,MAAA,CAAO,CAAA,GAAE,CAAF,KAAO,CAAd,EAAiB,WAAjB;AAFK,CAAN;;AAIA,QAAA,CAAS,QAAA,CAAA,CAAA;SACR,MAAA,CAAO,CAAA,GAAE,CAAF,KAAO,CAAd;AADQ,CAAT",
  "sourcesContent": [
    "# exceptions.test.coffee\n\nimport * as lib from '@jdeighan/base-utils/exceptions'\nObject.assign(global, lib)\nimport {fails, succeeds} from '@jdeighan/base-utils/utest'\n\n# ---------------------------------------------------------------------------\n\nfails () ->\n\tsuppressExceptionLogging true\n\tcroak(\"BAD\")\n\nfails () ->\n\tsuppressExceptionLogging true\n\tassert(2+2 != 4, 'EXCEPTION')\n\nsucceeds () ->\n\tassert(2+2 == 4)"
  ]
}